app/controllers/users_controller.rb

```diff
@@ -31,6 +31,7 @@ def destroy
     def edit
         @user = User.find(params[:id])
     end
+    
```

Comment:
please fix this error, this is because there is no `def update` in the `users_controller.rb`
![image](https://user-images.githubusercontent.com/73863408/128825033-cf8c8e9e-4bd9-48e6-8183-04cde0cd8554.png)